django/contrib/admin/static/admin/css/dark_mode.css

```
@@ -5,7 +5,8 @@
   
       --body-fg: #eeeeee;
       --body-bg: #121212;
-      --body-quiet-color: #e0e0e0;
+      --body-quiet-color: #989898;
+      --body-medium-color: #e0e0e0;
```

```suggestion
      --body-quiet-color: #d7d7d7;
      --body-medium-color: #e3e3e3;
```
Just to highlight that the quiet has become "quieter" and this darkens them both a bit to meet the AAA grade

---

Thank you!

Well spotted. I'm pushing a change with `#d0d0d0` for quiet (it ranks a bit better, `AAA` for `message-success-bg`), and I kept `#e0e0e0` because ranking remains the same (in terms of `AAA`) and it could potentially reduce UI changes (some elements that used to be `body-quiet-color` are now `body-medium-color`).